### DISCUSSION TITLE
Which type of 🍚 is most often used in 🍣?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#947] [Galaxy Trivia] [answered]
**Bonus:** When sushi was invented, what role did rice play?

**Comment by tomerf** [comment]
The type of rice most often used in sushi is short-grain Japanese rice. This type of rice has a sticky texture when cooked, which helps the sushi rolls hold their shape.
Modern sushi was invented in Japan around 1824. It originated around the 8th century in Southeast Asia as a method of preserving fish by fermenting it with rice. The rice acted as a natural preservative, preventing the fish from spoiling.

**Comment by peterbaccaro** [comment, marked as answer]
The type of rice most often used in sushi is called "**sushi rice**" or "**Japanese short-grain rice**". This rice is known for its sticky and slightly sweet texture, which helps the sushi ingredients hold together. Sushi rice is typically cooked with a specific ratio of water and rice vinegar, giving it a distinctive flavour and allowing it to better complement the flavours of the sushi fillings and toppings.

Rice played a crucial role in the invention of sushi. The origins of sushi can be traced back to ancient Southeast Asia, where people would ferment fish with **rice as a preservation method**. This early form of sushi was not consumed with the rice; instead, the **rice was used to preserve the fish and then discarded before eating**.

Over time, this preservation technique made its way to Japan around the 8th century, where the process evolved. The Japanese discovered that by using fermented rice to preserve fish, it not only extended the fish's shelf life but also enhanced its flavour. The rice acted as a natural fermentation agent, breaking down the proteins in the fish and imparting a tangy taste.

Eventually, the Japanese started consuming the fish along with the rice, creating the basis for what we now know as sushi. Over the centuries, the fermentation process was refined, and the rice was cooked rather than fermented. The rice was seasoned with vinegar to give it a pleasant taste and sticky texture, which helped bind the ingredients together.

Today, sushi rice remains a fundamental component of sushi, providing a base for various fillings, toppings, and rolls. Its sticky texture allows the sushi to hold its shape and makes it easier to eat with chopsticks or by hand. The vinegared flavour of the rice also complements the other ingredients, creating a harmonious balance of tastes in sushi.
